src/command/graph/push.rs

```diff
@@ -8,13 +6,16 @@ use rover_client::query::schema::push;
 
 use crate::client::get_studio_client;
 use crate::command::RoverStdout;
+use crate::utils::loaders::load_schema_from_flag;
+use crate::utils::parsers::{parse_schema_location, SchemaLocation};
 
 #[derive(Debug, Serialize, StructOpt)]
 pub struct Push {
-    /// Path of .graphql/.gql schema file to push
-    #[structopt(name = "SCHEMA_PATH", parse(from_os_str))]
+    /// The schema file to push
+    /// Can pass `-` to use stdin instead of a file
+    #[structopt(long, short = "s", parse(try_from_str = parse_schema_location))]
     #[serde(skip_serializing)]
-    schema_path: PathBuf,
+    schema: SchemaLocation,
```

Comment:
nit: might wanna call it `SchemaSource`? don't really care one way or another though `SchemaLocation` is def good enough

Reply:
I like that 👍